### PR TITLE
Bug 1249199 - modify replacement of locale string. r=rickychien

### PR DESCRIPTION
**Pull request by mantaroh**
https://bugzilla.mozilla.org/show_bug.cgi?id=1249199

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=152b1d50478eff01007bf6202d48b7ce5d0920ca

File: build/webapp-optimize.js [hunk 1]
```diff
@@ -661,7 +661,7 @@ function embedL10nResources(node, entries) {
 
 function getL10nJSONFileName(htmlFile, buildDirectoryFilePath) {
   var relativePath = utils.relativePath(buildDirectoryFilePath, htmlFile.path);
-  var base = relativePath.replace('.html', '').replace(/\//g, '.');
+  var base = relativePath.replace('.html', '').replace(/\//g, '.').replace(/\\/g, '.');
```
Comment (rickychien) [inline build/webapp-optimize.js]: jshint error: line too long.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ff52eed99cbacd989cf4830244d41d1bbebd8ecb

**Comment by mantaroh** [comment]
Oh sorry, I fixed the your comment.

**Pull request merged**